wagtail/snippets/templates/wagtailsnippets/snippets/type_index.html

```diff
@@ -40,7 +40,7 @@ <h1 class="w-header__title">
                     </div>
                 {% endif %}
             </div>
-            <div class="right col">
+            <div class="w-header__secondary col">
```

Comment:
This is similar to the classes in wagtail/admin/templates/wagtailadmin/shared/header_with_locale_selector.html

the `.col` addition for these divs may need to be solved in a more generic way or see whether we need this at all.

Maybe a class modifier on the root `w-header` can be used to adjust this behaviour on these sub-container elements.